src/pyodide.js

```diff
@@ -299,18 +299,14 @@ var languagePluginLoader = new Promise((resolve, reject) => {
   ////////////////////////////////////////////////////////////
   // Rearrange namespace for public API
   let PUBLIC_API = [
-    'globals',
-    'loadPackage',
-    'loadedPackages',
-    'pyimport',
-    'repr',
-    'runPython',
-    'runPythonAsync',
-    'checkABI',
-    'version',
-    'autocomplete',
+    'globals', 'loadPackage', 'loadedPackages', 'pyimport', 'repr', 'runPython',
+    'runPythonAsync', 'checkABI', 'version', 'autocomplete'
```

Comment:
Are those changes due to clang-format, maybe revert them as they are unrelated?

Reply:
Yeah it's a clang-format thing.

Reply:
OK, but that would mean that the lint is not working in CI? None of these lines was changed in this PR..

Reply:
I don't think it's the CI's fault, I think it was something local on my computer. Not sure.